com.unity.shadergraph/Documentation~/HD-Custom-Color-Node.md

```diff
@@ -0,0 +1,27 @@
+# Custom Color Node (HDRP)
+
+The Custom Color Node accesses the custom pass color buffer allocated by HDRP.
+
+## Render pipeline compatibility
+
+| **Node**       | **Universal Render Pipeline (URP)** | **High Definition Render Pipeline (HDRP)** |
+| -------------- | ----------------------------------- | ------------------------------------------ |
+| Custom Color Node | No                                  | Yes                                        |
+
+## Ports
+
+| Name        | Direction           | Type  | Binding | Description |
+|:------------ |:-------------|:-----|:---|:---|
+| **UV** | Input | Vector 4 | Screen Position | Sets the normalized screen coordinates to sample. |
+| **Output** | Output      |    Vector 4 | None | The value in the custom pass color buffer at the sampled coordinates |
```

Comment:
| **Output** | Output      |    Vector 4 | None | The value the custom pass color buffer contains at the sampled coordinates. |